src/io/perun.clj

```diff
@@ -26,6 +26,27 @@
   '[[org.pegdown/pegdown "1.6.0"]
     [circleci/clj-yaml "0.5.3"]])
 
+(deftask dump-meta
```

Comment:
I like this task. It's just for debugging right? So user can see what is in meta between each tasks